stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/common/extensions/Channel.kt

```diff
@@ -2,30 +2,35 @@ package io.getstream.chat.android.ui.common.extensions
 
 import android.content.Context
 import androidx.annotation.StringRes
+import io.getstream.chat.android.client.ChatClient
 import io.getstream.chat.android.client.extensions.getUsersExcludingCurrent
 import io.getstream.chat.android.client.models.Channel
 import io.getstream.chat.android.client.models.Message
+import io.getstream.chat.android.client.models.User
 import io.getstream.chat.android.ui.R
 import io.getstream.chat.android.ui.common.extensions.internal.isCurrentUser
 
 /**
  * Returns the channel name if exists, or the list of member names if the channel is distinct.
  *
- * @param devValue The resource identifier of a fallback string if the [Channel] object lacks
+ * @param context The context to load string resources.
+ * @param currentUser The currently logged-in user.
+ * @param fallback The resource identifier of a fallback string if the [Channel] object lacks
  * information to construct a valid display name string.
  *
  * @return The display name of the channel.
  */
 @JvmOverloads
 public fun Channel.getDisplayName(
```

Comment:
Same, passing the user explicitly here. At least this method can be unit tested now if the user is passed from the outside.

Reply:
Can we mark it with @InternalStreamChatApi?